src/components/About.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box, Button, Flex, Heading, Paragraph } from "theme-ui";
+
+export const About = () => {
+  return (
+    <Box>
+      <Heading>About</Heading>
+      <Paragraph variant="block">
```

Comment:
I don't think we have any `block` variant for paragraphs defined in the theme 🤔

Reply:
I will remove this variant and let you know.